### ISSUE TITLE
TextAppearance.Material.Widget.Button.Borderless.Colored not found. Android

### ISSUE THREAD

**Issue by Vincenzo1997** [maxs15/react-native-spinkit#104]
Hi, when I run ./gradlew assembleRelease I take this error:
error: resource android:style/TextAppearance.Material.Widget.Button.Borderless.Colored not found.
error: resource android:style/TextAppearance.Material.Widget.Button.Colored not found.
/Users/vincenzocarlino/.gradle/caches/transforms-1/files-1.1/appcompat-v7-27.1.1.aar/8eacfdd1903ee6ed1976c5f0512e0f80/res/values-v26/values-v26.xml:9:5-12:13: AAPT: error: resource android:attr/colorError not found.
    
/Users/vincenzocarlino/.gradle/caches/transforms-1/files-1.1/appcompat-v7-27.1.1.aar/8eacfdd1903ee6ed1976c5f0512e0f80/res/values-v26/values-v26.xml:13:5-16:13: AAPT: error: resource android:attr/colorError not found.
    
/Users/vincenzocarlino/.gradle/caches/transforms-1/files-1.1/appcompat-v7-27.1.1.aar/8eacfdd1903ee6ed1976c5f0512e0f80/res/values-v26/values-v26.xml:17:5-93: AAPT: error: style attribute 'android:attr/keyboardNavigationCluster' not found.
    
/Users/vincenzocarlino/.gradle/caches/transforms-1/files-1.1/appcompat-v7-27.1.1.aar/8eacfdd1903ee6ed1976c5f0512e0f80/res/values/values.xml:251:5-69: AAPT: error: resource android:attr/fontStyle not found.
    
/Users/vincenzocarlino/.gradle/caches/transforms-1/files-1.1/appcompat-v7-27.1.1.aar/8eacfdd1903ee6ed1976c5f0512e0f80/res/values/values.xml:251:5-69: AAPT: error: resource android:attr/font not found.
    
/Users/vincenzocarlino/.gradle/caches/transforms-1/files-1.1/appcompat-v7-27.1.1.aar/8eacfdd1903ee6ed1976c5f0512e0f80/res/values/values.xml:251:5-69: AAPT: error: resource android:attr/fontWeight not found.
    
error: failed linking references.

Plase HELP !!

**Comment by AilsonFreire** [comment]
':react-native-spinkit:verifyReleaseResources'.
> java.util.concurrent.ExecutionException: com.android.builder.internal.aapt.v2.Aapt2Exception: Android resource linking failed

**Comment by kraksin** [comment]
Have you found a solution?
I really wonder ...

**Comment by yulolimum** [comment]
You want to increase the SDK version within the package's `build.gradle` (stackoverflow link for reference: https://stackoverflow.com/a/49332191) to be at least 24. In fact, there's a couple PRs open to do so (example: https://github.com/maxs15/react-native-spinkit/issues/104).

I used the `patch-package` library to patch this library to match the versions in my `android/build.gradle`. In my case, i changed it to 28:

```
diff --git a/node_modules/react-native-spinkit/android/build.gradle b/node_modules/react-native-spinkit/android/build.gradle
index 967ce17..0dbd0e6 100644
--- a/node_modules/react-native-spinkit/android/build.gradle
+++ b/node_modules/react-native-spinkit/android/build.gradle
@@ -20,8 +20,8 @@ buildscript {
 }
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion 28
+    buildToolsVersion "28.0.3"
 
     defaultConfig {
         minSdkVersion 16

```

**Comment by kraksin** [comment]
@yulolimum Thank you. I solved it before you wrote it, but I also resolved it by uploading the version.

**Comment by maxs15** [comment]
Fix available in 1.3.0
